lib/integration_whitelist.rb

```diff
@@ -0,0 +1,113 @@
+INTEGRATION_WHITELIST = [
```

Comment:
is the name of the directory in `integrations-core` always the same name as what `ddev` shows?

Reply:
I just used this function https://github.com/DataDog/integrations-core/blob/d18461375c85522e8bdf87fb67eef7740f151c0f/datadog_checks_dev/datadog_checks/dev/tooling/utils.py#L105 which prints the path of the integration, so the answer should be yes.